src/.gitlab-ci.yml.jinja

```diff
@@ -418,6 +421,8 @@ review_preprod:
   needs:
     - job: "update_db"
       optional: true
+    - job: "publish_db"
```

Comment:
it's the opposit no?
publish after review_preprod

Reply:
Hmm yes probably